docs/zkapps/how-to-write-a-zkapp.mdx

```diff
@@ -916,7 +916,9 @@ To summarize, there are three types of authorization that account updates can ha
 
 The list above just reflects common defaults. The full source of truth is given by the _account permissions_, see [Permissions](snarkyjs-reference/Interfaces/Permissions). Using permissions, account owners can decide on a fine-grained level which type of authorization is required on which kinds of updates.
 
-<!-- TODO: link to advanced permissions section instead of just API reference; this section should list all permissions, explain how permissions are changed through zkApp deployment, explain how accounts can be made non-upgradeable, and explain the `access` permission for the case of token contracts -->
+If you are looking for an advanced introduction to Permissions, take a look at [this](advanced-snarkyjs/permissions) section!
```

Comment:
This feels a bit redundant, I would merge this sentence into the paragraph above. For example

```diff
- The full source of truth is given by the _account permissions_, see [Permissions](snarkyjs-reference/Interfaces/Permissions).
+ The full source of truth is given by the _account permissions_, see [our in-depth section about Permissions](advanced-snarkyjs/permissions).
Using permissions, account owners can decide on a fine-grained level which type of authorization is required on which kinds of updates.

- If you are looking for an advanced introduction to Permissions, take a look at [this](advanced-snarkyjs/permissions) section!
```

Also IMO marking links with a generic small word like [this](https://github.com/o1-labs/docs2/pull/277/files) should be avoided